src/pages/Lending/Pool/components/Repay/RepayConfirm.tsx

```diff
@@ -54,40 +250,85 @@ export const RepayConfirm = () => {
             </Row>
+            <Skeleton isLoaded={!isLendingQuoteCloseLoading}>
+              <Row>
+                <Row.Label>{translate('common.receive')}</Row.Label>
+                <Row.Value textAlign='right'>
+                  <Stack spacing={1} flexDir='row' flexWrap='wrap'>
+                    <Amount.Crypto
+                      value={
+                        lendingQuoteCloseData?.quoteLoanCollateralDecreaseCryptoPrecision ?? '0'
+                      }
+                      symbol={repaymentAsset?.symbol ?? ''}
+                    />
+                    <Amount.Fiat
+                      color='text.subtle'
+                      value={
+                        lendingQuoteCloseData?.quoteLoanCollateralDecreaseFiatUserCurrency ?? '0'
+                      }
+                      prefix='≈'
+                    />
+                  </Stack>
+                </Row.Value>
+              </Row>
+            </Skeleton>
+            <Skeleton isLoaded={!isLendingQuoteCloseLoading}>
+              <Row fontSize='sm' fontWeight='medium'>
+                <HelperTooltip label='tbd'>
+                  <Row.Label>{translate('common.feesPlusSlippage')}</Row.Label>
+                </HelperTooltip>
+                <Row.Value>
+                  <Amount.Fiat
+                    value={lendingQuoteCloseData?.quoteTotalFeesFiatUserCurrency ?? '0'}
+                  />
+                </Row.Value>
+              </Row>
+            </Skeleton>
             <Row fontSize='sm' fontWeight='medium'>
               <Row.Label>{translate('common.gasFee')}</Row.Label>
               <Row.Value>
-                <Amount.Fiat value='10' />
+                <Amount.Fiat value='TODO' />
               </Row.Value>
             </Row>
           </Stack>
-          {/* @ts-ignore TODO(gomes): implement me */}
-          <LoanSummary borderTopWidth={0} mt={0} />
+          <LoanSummary
+            repaymentAsset={repaymentAsset}
+            collateralAssetId={collateralAssetId}
+            repaymentPercent={repaymentPercent ?? 0}
+            repayAmountCryptoPrecision={repaymentAmountCryptoPrecision ?? '0'}
+            collateralDecreaseAmountCryptoPrecision={
+              lendingQuoteCloseData?.quoteLoanCollateralDecreaseCryptoPrecision ?? '0'
+            }
+            repaymentAccountId={repaymentAccountId}
+            collateralAccountId={collateralAccountId}
+            debtRepaidAmountUsd={lendingQuoteCloseData?.quoteDebtRepaidAmountUsd ?? '0'}
+            borderTopWidth={0}
+            mt={0}
+          />
           <CardFooter px={4} py={4}>
-            <Button colorScheme='blue' size='lg' width='full'>
+            <Button
+              isLoading={isLoanClosePending}
+              disabled={isLoanClosePending}
+              onClick={handleRepay}
+              colorScheme='blue'
+              size='lg'
+              width='full'
+            >
               {translate('lending.confirmAndBorrow')}
```

Comment:
We'll want to replace this will `confirmAndRepay` or something.

Reply:
db2c609dcd